### PR TITLE
BootLintBear: Move to HTML section

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
The BootLintBear is now in the HTML folder.

Fixes https://github.com/coala-analyzer/coala-bears/issues/158

### CODE REVIEW

**Comment by AbdealiJK** [comment]
a1d6e9e unack

Need to move the tests ?

File: requirements.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+restructuredtext_lint~=0.14.0
+proselint~=0.3.5
+cpplint~=0.0.7
+isort~=4.2.2
+html-linter~=0.3.0
 # Do *not* use 0.3 which is not backwards compatible to common clang versions
 libclang-py3==0.2
-guess-language-spirit>=0.5.2
+guess-language-spirit~=0.5.2
 radon==1.2.2
```
Comment (AbdealiJK) [inline requirements.txt]: why not radon ?

File: requirements.txt [hunk 2]
```diff
@@ -1,23 +1,23 @@
 coala==0.4.2.dev20160316074840
```
Comment (AbdealiJK) [inline requirements.txt]: coala too
@sils1297 how do we handle this ? We first move coala to 0.5.0 and then change this to ~0.5.0 ?

**Comment by Adrianzatreanu** [comment]
tests are moved now too. anything else i missed?

**Comment by sils1297** [comment]
if tests pass https://github.com/coala-analyzer/coala-bears/pull/250/commits/39e72595dd0f6e2908248b6535f422d71eb8ce6e is ready

**Comment by Adrianzatreanu** [comment]
@sils1297 there's one more commit here that i sent now cuz i was rebasing earlier.

File: requirements.txt (hunk 2 above)
Comment (sils1297) [inline requirements.txt]: when we release 0.5.0 we switch this to `~0.5.0`

File: requirements.txt (hunk 1 above)
Comment (sils1297) [inline requirements.txt]: because radon uses a == that broke our requirements IIRC

Comment (AbdealiJK) [inline requirements.txt]: Then we need a comment here saying that

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/250/commits/f8c9f89f5ba0d910e24367a432264faf1a8901cc, more coming after coala-analyzer/PyPrint#34 is merged.

File: circle.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ test:
         parallel: true
     - python setup.py install:
         parallel: true
-    - coala-ci -S=autoapply=false:
+    - coala-ci -L DEBUG:
```
Comment (sils1297) [inline circle.yml]: this is unrelated

File: .ci/deps.sh [hunk 4]
```diff
@@ -81,3 +81,6 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
 	opam install --deps-only --yes infer
 	./build-infer.sh java
 fi
+
+# Upgrade pip to make sure PEP440 is supported i. e. ~= for versioning
+pip install -U pip
```
Comment (SanketDG) [inline .ci/deps.sh]: would be better if you do it at the top, to avoid any future problems.

File: circle.yml [hunk 5]
```diff
@@ -1,4 +1,41 @@
 dependencies:
+<<<<<<< HEAD
```
Comment (SanketDG) [inline circle.yml]: you might wanna remove these.

File: .ci/deps.sh [hunk 6]
```diff
@@ -1,6 +1,8 @@
 set -e
 set -x
 
+pip install -U pip
```
Comment (sils1297) [inline .ci/deps.sh]: shouldn't be needed

File: .ci/deps.sh [hunk 7]
```diff
@@ -71,7 +73,7 @@ julia -e "Pkg.add(\"Lint\")"
 sudo luarocks install luacheck --deps-mode=none
 
 # Infer commands
-if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then 
+if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
```
Comment (sils1297) [inline .ci/deps.sh]: unrelated

Comment (sils1297) [inline .ci/deps.sh]: can you please check your commits before you submit them for review?

File: .ci/deps.sh (hunk 6 above)
Comment (SanketDG) [inline .ci/deps.sh]: why not?

Comment (SanketDG) [inline .ci/deps.sh]: circle is failing it seems

**Comment by sils1297** [comment]
moving to https://github.com/coala-analyzer/coala-bears/pull/254
